bigbone/src/integrationTest/kotlin/social/bigbone/v410/V410StatusMethodsIntegrationTest.kt

```diff
@@ -311,4 +305,300 @@ class V410StatusMethodsIntegrationTest {
+
+            // then
+            assertFalse(unfavouritedStatus.isFavourited)
+        }
+    }
+
+    @Nested
+    @DisplayName("pinStatus tests")
+    internal inner class PinStatusTests {
+        @Test
+        fun `should pin status when all params set`() {
+            // given
+            val user1Client = TestHelpers.getTrustAllClient(user1UserToken.accessToken)
+            val statusId = user1Client.statuses.postStatus(status = "This post will be pinned soon").execute().id
+            TestHelpers.unpinAllPinnedStatuses(user1Client)
+
+            // when
+            val pinnedStatus = user1Client.statuses.pinStatus(statusId).execute()
+
+            // then
+            assertTrue(pinnedStatus.isPinned!!)
+        }
+    }
+
+    @Nested
+    @DisplayName("unpinStatus tests")
+    internal inner class UnpinStatusTests {
+        @Test
+        fun `should unpin status when all params set`() {
+            // given
+            val user1Client = TestHelpers.getTrustAllClient(user1UserToken.accessToken)
+            val statusId = user1Client.statuses.postStatus(status = "This post will be pinned soon, then unpinned again").execute().id
+            TestHelpers.unpinAllPinnedStatuses(user1Client)
+            user1Client.statuses.pinStatus(statusId).execute()
+
+            // when
+            val unpinnedStatus = user1Client.statuses.unpinStatus(statusId).execute()
+
+            // then
+            assertFalse(unpinnedStatus.isPinned!!)
+        }
+    }
+
+    @Nested
+    @DisplayName("muteConversation tests")
+    internal inner class MuteConversationTests {
+        @Test
+        fun `should mute conversation when all params set`() {
+            // given
+            val user1Client = TestHelpers.getTrustAllClient(user1UserToken.accessToken)
+            val user2Client = TestHelpers.getTrustAllClient(user2UserToken.accessToken)
+            val statusId = user1Client.statuses.postStatus(status = "This conversation will be muted soon").execute().id
+
+            // when
+            val mutedConversation = user2Client.statuses.muteConversation(statusId).execute()
+
+            // then
+            assertTrue(mutedConversation.isMuted)
+        }
+    }
+
+    @Nested
+    @DisplayName("unmuteConversation tests")
+    internal inner class UnmuteConversationTests {
+        @Test
+        fun `should unmute conversation when all params set`() {
+            // given
+            val user1Client = TestHelpers.getTrustAllClient(user1UserToken.accessToken)
+            val user2Client = TestHelpers.getTrustAllClient(user2UserToken.accessToken)
+            val statusId = user1Client.statuses.postStatus(status = "This conversation will be muted soon, then unmuted again").execute().id
+            user2Client.statuses.muteConversation(statusId).execute()
+
+            // when
+            val unmutedConversation = user2Client.statuses.unmuteConversation(statusId).execute()   //.pinStatus(statusId).execute()
```

Comment:
commented out code at the end of the line

Reply:
Fixed!